### ISSUE TITLE
emptyLineBefore not working?

### ISSUE THREAD

**Issue by nickngqs** [hudochenkov/postcss-sorting#59]
Recently my emptyLineBefore is not working. I didn't change any config, it just stop working. I looked up the release but can't find any changes that would affect this.

```
{
	"clean-empty-lines": true,
	"sort-on-save": true,
	"order": [
		"custom-properties",
		"dollar-variables",
		{
			"type": "at-rule",
			"name": "include",
			"parameter": "margin"
		},
		{
			"type": "at-rule",
			"name": "include",
			"parameter": "padding"
		},
		"declarations",
		{
			"type": "at-rule",
			"name": "include",
			"parameter": "grid-container"
		},
		{
			"type": "at-rule",
			"name": "include",
			"parameter": "grid-collapse"
		},
		{
			"type": "at-rule",
			"name": "include",
			"parameter": "grid-column"
		},
		{
			"type": "at-rule",
			"name": "include",
			"parameter": "grid-width"
		},
		"at-rules",
		"rules",
		{
			"type": "at-rule",
			"name": "include",
			"parameter": "media"
		}
	],
	"properties-order": [
		{
			"emptyLineBefore": true,
			"properties": [
				"display",
				"align-items",
				"align-content",
				"justify-content",
				"flex-direction",
				"flex-order",
				"flex-pack",
				"flex-align",
				"visibility",
				"float",
				"clear",
				"overflow",
				"overflow-x",
				"overflow-y",
				"clip",
				"zoom"
			]
		},
		{
			"emptyLineBefore": true,
			"properties": [
				"position",
				"top",
				"right",
				"bottom",
				"left",
				"z-index"
			]
		},
		{
			"emptyLineBefore": true,
			"properties": [
				"box-sizing",
				"width",
				"min-width",
				"max-width",
				"height",
				"min-height",
				"max-height",
				"margin",
				"margin-top",
				"margin-bottom",
				"margin-left",
				"margin-right",
				"padding",
				"padding-top",
				"padding-bottom",
				"padding-left",
				"padding-right"
			]
		},
		{
			"emptyLineBefore": true,
			"properties": [
				"font",
				"font-family",
				"font-size",
				"font-weight",
				"font-style",
				"font-variant",
				"font-size-adjust",
				"font-stretch",
				"font-effect",
				"font-emphasize",
				"font-emphasize-position",
				"font-emphasize-style",
				"font-smooth",
				"line-height",
				"letter-spacing",
				"color"
			]
		},
		{
			"emptyLineBefore": true,
			"properties": [
				"table-layout",
				"empty-cells",
				"caption-side",
				"border-spacing",
				"border-collapse",
				"list-style",
				"list-style-position",
				"list-style-type",
				"list-style-image"
			]
		},
		{
			"emptyLineBefore": true,
			"properties": [
				"content",
				"quotes",
				"counter-reset",
				"counter-increment",
				"resize",
				"cursor",
				"user-select",
				"nav-index",
				"nav-up",
				"nav-right",
				"nav-down",
				"nav-left",
				"text-align",
				"text-align-last",
				"vertical-align",
				"white-space",
				"text-decoration",
				"text-emphasis",
				"text-emphasis-color",
				"text-emphasis-style",
				"text-emphasis-position",
				"text-indent",
				"text-justify",
				"word-spacing",
				"writing-mode",
				"text-outline",
				"text-transform",
				"text-wrap",
				"text-overflow",
				"text-overflow-ellipsis",
				"text-overflow-mode",
				"word-wrap",
				"word-break",
				"tab-size",
				"hyphens",
				"pointer-events"
			]
		},
		{
			"emptyLineBefore": true,
			"properties": [
				"opacity",
				"filter",
				"interpolation-mode",
				"border",
				"border-width",
				"border-style",
				"border-color",
				"border-top",
				"border-top-width",
				"border-top-style",
				"border-top-color",
				"border-right",
				"border-right-width",
				"border-right-style",
				"border-right-color",
				"border-bottom",
				"border-bottom-width",
				"border-bottom-style",
				"border-bottom-color",
				"border-left",
				"border-left-width",
				"border-left-style",
				"border-left-color",
				"border-radius",
				"border-top-left-radius",
				"border-top-right-radius",
				"border-bottom-right-radius",
				"border-bottom-left-radius",
				"border-image",
				"border-image-source",
				"border-image-slice",
				"border-image-width",
				"border-image-outset",
				"border-image-repeat",
				"outline",
				"outline-width",
				"outline-style",
				"outline-color",
				"outline-offset",
				"background",
				"background-color",
				"background-image",
				"background-repeat",
				"background-attachment",
				"background-position",
				"background-position-x",
				"background-position-y",
				"background-clip",
				"background-origin",
				"background-size",
				"box-decoration-break",
				"box-shadow",
				"text-shadow"
			]
		}, 
		{
			"emptyLineBefore": true,
			"properties": [
				"transition",
				"transition-delay",
				"transition-timing-function",
				"transition-duration",
				"transition-property",
				"transform",
				"transform-origin",
				"animation",
				"animation-name",
				"animation-duration",
				"animation-play-state",
				"animation-timing-function",
				"animation-delay",
				"animation-iteration-count",
				"animation-direction"
			]
		}
	]
}
```

**Comment by hudochenkov** [comment]
Could you, please, provide a stylesheet where `emptyLineBefore` doesn't work?

What version do you use?
How do you use postcss-sorting (via gulp-postcss, text editor plugin, etc.)?

**Comment by nickngqs** [comment]
I'm using PossCSSSorting v.2.1.0 in Sublime Text plugin.

```
fieldset {
  margin: 0;
  padding: 0;

  border: 0;
  background-color: transparent;
}
```
It used to add emptylines before group

Now it collapses them
```
fieldset {
  margin: 0;
  padding: 0;
  border: 0;
  background-color: transparent;
}
```

**Comment by hudochenkov** [comment]
Thanks you for reporting!

It was a problem with `sublime-postcss-sorting`. I released new version `2.1.1`. It should be available in Package Control in an hour or two.
